ipysurfer/viz/brain.py

```diff
@@ -0,0 +1,541 @@
+        Specifies how the cortical surface is rendered. Options:
+
+            1. The name of one of the preset cortex styles:
+            ``'classic'`` (default), ``'high_contrast'``,
+            ``'low_contrast'``, or ``'bone'``.
+            2. A color-like argument to render the cortex as a single
+            color, e.g. ``'red'`` or ``(0.1, 0.4, 1.)``. Setting
+            this to ``None`` is equivalent to ``(0.5, 0.5, 0.5)``.
+            3. The name of a colormap used to render binarized
+            curvature values, e.g., ``Grays``.
+            4. A list of colors used to render binarized curvature
+            values. Only the first and last colors are used. E.g.,
+            ['red', 'blue'] or [(1, 0, 0), (0, 0, 1)].
+            5. A container with four entries for colormap (string
+            specifying the name of a colormap), vmin (float
+            specifying the minimum value for the colormap), vmax
+            (float specifying the maximum value for the colormap),
+            and reverse (bool specifying whether the colormap
+            should be reversed. E.g., ``('Greys', -1, 2, False)``.
+            6. A dict of keyword arguments that is passed on to the
+            call to surface.
+    alpha : float in [0, 1]
+        Alpha level to control opacity of the cortical surface.
+    size : float or pair of floats
+        the size of the window, in pixels. can be one number to specify
+        a square window, or the (width, height) of a rectangular window.
+    background : matplotlib color
+        Color of the background.
+    foreground : matplotlib color
+        Color of the foreground (will be used for colorbars and text).
+        None (default) will use black or white depending on the value
+        of ``background``.
+    figure : list of mayavi.core.scene.Scene | None | int
+        If None (default), a new window will be created with the appropriate
+        views. For single view plots, the figure can be specified as int to
+        retrieve the corresponding Mayavi window.
+    subjects_dir : str | None
+        If not None, this directory will be used as the subjects directory
+        instead of the value set using the SUBJECTS_DIR environment
+        variable.
+    views : list | str
+        views to use.
+    offset : bool
+        If True, aligs origin with medial wall. Useful for viewing inflated
+        surface where hemispheres typically overlap (Default: True).
+    show_toolbar : bool
+        If True, toolbars will be shown for each view.
+    offscreen : bool
+        If True, rendering will be done offscreen (not shown). Useful
+        mostly for generating images or screenshots, but can be buggy.
+        Use at your own risk.
+    interaction : str
+        Can be "trackball" (default) or "terrain", i.e. a turntable-style
+        camera.
+    units : str
+        Can be 'm' or 'mm' (default).
+
+    Attributes
+    ----------
+    geo : dict
+        Cortex surface objects.
```

Comment:
clarify if these are ipyvolume objects or something unique to ipysurfer

Reply:
ok